### DISCUSSION TITLE
Configure vim-visual-multi in NvChad

### DISCUSSION THREAD

**Discussion by abhishekghimire40** [NvChad/NvChad#2357] [General]
I am  trying to configure vim-visual-multi in nvchad but i am unable to do so. 
This is what i have added in my custom plugins file:
{
	"mg979/vim-visual-multi",
	branch = "master",
	init = function()
		vim.g.VM_maps = { 
                    ["Find Under"] = "<C-n>"
                }
	end,
}
But everytime i use <C-n> it jumps to a newline. I have also tried to use other keymappings but it is not working. 
Can anyone help me configure it?

**Comment by siduck** [comment]
event="VeryLazy"

add that below branch spec

**Reply by abhishekghimire40** [reply]
It worked.Thank you
